django/utils/datastructures.py

```diff
@@ -285,6 +285,9 @@ def __getitem__(self, key):
 
 
 def _destruct_iterable_mapping_values(data):
```

Comment:
So the last thing I'd suggest is moving this to be a `@staticmethod` on `CaseInsensitiveMapping`, and rename it to something like `_unpack_values()`.

This means that we don't need to import the separate private function into `django.http.response` and can call `self.unpack_values(data)`.

Reply:
Done. I named it `_unpack_items` though :)